### PR TITLE
Fixes for SF3.4

### PR DESCRIPTION
**Pull request by erik-zicht**
TODO: create a new releasebranch when approved and update README and CHANGELOG

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,10 +11,10 @@
     ],
     "require": {
         "php": "^7",
-        "symfony/symfony": "^2.7",
+        "symfony/symfony": "^3.4",
         "doctrine/doctrine-bundle": "^1",
         "sonata-project/doctrine-orm-admin-bundle": "^3 || ^2",
-        "zicht/framework-extra-bundle": "^4.1 || ^5",
+        "zicht/framework-extra-bundle": "4 - 6",
```
Comment (pbergman) [inline composer.json]: should be >= 6, others wont support php7, perhaps wise to check other dependencies?

File: src/Zicht/Bundle/AdminBundle/Security/Voter/AdminVoter.php [hunk 2]
```diff
@@ -41,11 +45,12 @@ class AdminVoter implements VoterInterface
      * @param ContainerInterface $serviceContainer
      * @param array $attributes
      */
-    public function __construct(Pool $pool, ContainerInterface $serviceContainer, array $attributes)
+    public function __construct(Pool $pool, ContainerInterface $serviceContainer, array $attributes, AccessDecisionManagerInterface $decisionManager)
```
Comment (pbergman) [inline src/Zicht/Bundle/AdminBundle/Security/Voter/AdminVoter.php]: perhaps better to remove the whole dependency on `ContainerInterface`?

File: src/Zicht/Bundle/AdminBundle/Security/Handler/RoleSecurityHandler.php [hunk 3]
```diff
@@ -20,17 +20,8 @@ class RoleSecurityHandler extends BaseRoleSecurityHandler
...
-            if ($this->securityContext->isGranted($attributes, $object)) {
-                return true;
-            }
-        }
-        // sonata admin 3
-        if (isset($this->authorizationChecker)) {
-            if ($this->authorizationChecker->isGranted($attributes, $object)) {
-                return true;
-            }
+        if ($this->authorizationChecker->isGranted($attributes, $object)) {
```
Comment (pbergman) [inline src/Zicht/Bundle/AdminBundle/Security/Handler/RoleSecurityHandler.php]: i think you still got dependency on old version in composer file.

**Review by pbergman — CHANGES_REQUESTED** [review]

File: src/Zicht/Bundle/AdminBundle/Security/Handler/RoleSecurityHandler.php (hunk 3 above)
Comment (erik-zicht) [reply src/Zicht/Bundle/AdminBundle/Security/Handler/RoleSecurityHandler.php]: Fixed by dropping support for v2 of doctrine-orm-admin-bundle

**Review by pbergman — APPROVED** [review]

**Pull request merged**